compose.override.yaml

```diff
@@ -6,6 +6,7 @@ services:
       target: frankenphp_dev
     volumes:
       - ./:/app
+      - /app/var
```

Comment:
Are you sure for this ? Or should we put it at the same level as the `#- app/vendor` line just below ?

Reply:
What I mean is that I personally treat `var` & `vendor` in the same way on my projects.

Reply:
Sorry, this was a leftover of an unrelated experiment. Fixed.